### ISSUE TITLE
consider removing `exercism-tooling` topic exercism-wide

### ISSUE THREAD

**Issue by ee7** [exercism/org-wide-files#206]
It seems that the set of repos with the `exercism-tooling` topic is exactly the set of repos with `exercism-test-runner` or `exercism-representer` or `exercism-analyzer`.

So would it be better to remove `exercism-tooling` everywhere, and check one of those other topics is present?

Pros that I can see:
1. It would prevent the two sets from drifting apart, which might cause e.g. some repo to be omitted by some change. Evidence that topics are somewhat error-prone already: I mentioned to Erik that [exercism/abap-test-runner](https://github.com/exercism/abap-test-runner) was missing its topics. 
1. `exercism-tooling` is an unclear name: to an outsider, it is not obvious which repos should have the `exercism-tooling` topic. Someone might wonder why it's missing from, say, `exercism/org-wide-files`, `exercism/github-actions`, or `exercism/configlet`. We can't add it, because then @exercism-bot will create PRs that add `deploy.yml`.
1. It would help the problem in https://github.com/exercism/nim-docker-base/pull/12. A base docker image that is used by a test-runner is "exercism tooling", but it shouldn't currently have the `exercism-tooling` topic.
1. We could then more reasonably add a new topic to group "tooling repos" that are not test runners, representers, or analyzers, which currently do not have a topic.

Cons that I can see:
1. Somebody can't go to https://github.com/search?q=org%3Aexercism+topic%3Aexercism-tooling to see every test runner, representer, and analyzer repo (assuming that the topic is correctly applied).
1. Time to implement.

It looks like the `exercism-tooling` topic is only referenced in workflows in `org-wide-files`, and some local scripts on the machines of people who would be notified of this issue. So maybe it's worth removing it.

**Comment by SaschaMann** [comment]
The topics have been appropriated for the use in the org-wide-files flows. I'd argue we should lean towards changing org-wide-files rather than changing the topic. That said, I'm not sure if anyone uses topics for anything else so it might not be a problem.

**Comment by SaschaMann** [comment]
> I didn't understand this. Could you rephrase?

I meant to say that if the way `org-wide-files` interprets that topic prevents the topic from being applied to a repo, e.g. nim-docker-base, we should change `org-wide-files`.
